client/src/app/site/motions/components/workflow-detail/workflow-detail.component.ts

```diff
@@ -200,11 +200,13 @@ export class WorkflowDetailComponent extends BaseViewComponent implements OnInit
      * Opens a dialog to enter the workflow name
      */
     public onNewStateButton(): void {
-        this.openEditDialog('', 'Create new state', 'Name').subscribe(result => {
-            if (result && result.action === 'update') {
-                this.workflowRepo.addState(result.value, this.workflow).then(() => {}, this.raiseError);
+        this.openEditDialog('', this.translate.instant('Create new state'), this.translate.instant('Name')).subscribe(
```

Comment:
Has translation already. Avoid double declaring translation, especially using `.instant`

https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/site/motions/components/workflow-detail/workflow-detail.component.html#L115

https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/site/motions/components/workflow-detail/workflow-detail.component.html#L118

Reply:
This doen't work, too. You have to mark the specific string before the template can translate it.

Reply:
Why not using the marker function like in https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/core/marked-translations.ts#L1

(even tho that was just copying the content of ngx-translate-extract, that should not be commented out btw)